src/main.test.js

```diff
@@ -12,7 +12,7 @@ test("Valid input with type", () => {
 
 test("Valid input with restricted choices", () => {
   const testEnv = "DATABASE_URL\nNUMBER :[0,1]";
-  expect(envSpecToHTML(testEnv)).toEqual(
+  expect(envSpec.parse(testEnv)).resolves.toEqual(
```

Comment:
You need to `return` this, in order to mark the test successfully as passed/failing ([see example](https://jestjs.io/docs/en/expect.html#resolves)).